Containerfile

```diff
@@ -1,5 +1,5 @@
 FROM registry.fedoraproject.org/fedora:39 AS builder
-RUN dnf install -y git-core golang gpgme-devel libassuan-devel && mkdir -p /build/bib
+RUN dnf install -y git-core golang gpgme-devel libassuan-devel device-mapper-devel && mkdir -p /build/bib
```

Comment:
I think we can safely not care about `devicemapper` FWIW.  But, no objections from me for including it either.

Reply:
Yes remove it. We don't want it and no reason to add it to the container.  Not sure we want BTRFS either.